derp/derp_server.go

```diff
@@ -1955,6 +1955,22 @@ func (s *Server) ConsistencyCheck() error {
 			s.curClients.Value(),
 			len(s.clients)))
 	}
+	if s.verifyClientsLocalTailscaled {
+		ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+		defer cancel()
+		status, err := localClient.Status(ctx)
+		if err != nil {
+			errs = append(errs, fmt.Sprintf("localClient.Status: %v", err))
+		}
+		info := &clientInfo{
+			IsProber: true,
+		}
+		clientIP := netip.IPv6Loopback()
+		if err := s.verifyClient(ctx, status.Self.PublicKey, info, clientIP); err != nil {
```

Comment:
Probably want to put this in an `else if` or you'll get a nil pointer exception if `Status` fails

Reply:
Moved out to a method